src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -57,10 +63,14 @@ public JsonAdaptedPerson(Person source) {
         phone = source.getPhone().value;
         email = source.getEmail().value;
         address = source.getAddress().value;
+        remark = source.getRemark().value;
+        birthday = Optional.ofNullable(source.getBirthday().value)
```

Comment:
Optional is used only in editPersonDescriptor since edit command may leave some field unchanged, in Person class there is no Optional usage, it must be present with either a legit value or the placeholder singleton

Reply:
Ok, fixed it. I realised a much shorter way around this - use Birthday's toString()